source/PCA9685Support.cpp

```diff
@@ -458,16 +454,6 @@ HRESULT PCA9685Device::_InitializeChip(ULONG i2cAdr)
 
         if (SUCCEEDED(hr))
         {
-            // Delay for 500 microseconds for clock to start.
```

Comment:
This delay is required to allow the internal clock to start when the chip is taken out of sleep.